.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Release
+on:
+  release:
+    types:
+      - created
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - run: ./ciScripts/buildPlugin.sh
+        env:
+          RELEASE_NOTES: ${{ github.event.release.body }}
+          VERSION: ${{ github.ref }}
+      - name: Copy zip file
+        run: cp build/distributions/*.zip waifu-motivator.zip
+      - uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: waifu-motivator.zip
+          asset_name: waifu-motivator.zip
+          asset_content_type: application/zip
+      - run: ./gradlew publishPlugin
+        env:
+          VERSION: ${{ github.ref }}
+          # empty channel for default
+          PUBLISH_CHANNEL:
+          PUBLISH_TOKEN: ${{ secrets.PUBLISH_TOKEN }}
```

Comment:
You'll need to set that up, as I can't do that.

Reply:
Added

<img width="963" alt="Screen Shot 2020-09-08 at 07 01 03" src="https://user-images.githubusercontent.com/21978370/92419603-1bffd080-f1a1-11ea-9312-5b751bbb6d5b.png">